Documentation/contributing/development/introducing_new_crds.rst

```diff
@@ -0,0 +1,400 @@
+Introducing new CRDs
```

Comment:
Please choose between title case or regular case and stick to it for all titles in the page. I think title case is used more often in the docs.